tools/ci/actions/configure.sh

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env bash
+#**************************************************************************
+#*                                                                        *
+#*                                 OCaml                                  *
+#*                                                                        *
+#*                 David Allsopp, OCaml Labs, Cambridge.                  *
+#*                                                                        *
+#*   Copyright 2021 David Allsopp Ltd.                                    *
+#*                                                                        *
+#*   All rights reserved.  This file is distributed under the terms of    *
+#*   the GNU Lesser General Public License version 2.1, with the          *
+#*   special exception on linking described in the file LICENSE.          *
+#*                                                                        *
+#**************************************************************************
+
+#------------------------------------------------------------------------
+#This test checks that if configure.ac has been modified by the pull
+#request, then configure has been correctly regenerated.
+#------------------------------------------------------------------------
```

Comment:
(same question on comments vs. messages.)

Reply:
There was already a message for this in the original script, which I've enhanced further - cf. https://github.com/dra27/ocaml/runs/1865755066?check_suite_focus=true